### PR TITLE
CORE-1799: correct CPU usage updates sent to QMS

### PR DESCRIPTION
**Pull request by slr71**
I still have some more testing to do on this, but I think it's ready for at least a cursory review.

Prior to this change, the cumulative CPU usage total since `resource-usage-api` was moved to production was being sent to QMS as an incremental change, resulting in inflated CPU usage totals. (Note: this isn't the only problem with CPU usage updates; the other problem is in `QMS`.) The purpose of this change is to modify `resource-usage-api` to send the correct incremental update amounts to `QMS`.

### CODE REVIEW

File: internal/nats.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	}
+
+	// Update the logger to include the username.
+	log = log.WithField("username", username)
+	log.Debugf("username for user ID %s is %s", userID, username)
+
+	// QMS currently only supports incremental updates.
+	if usageEvent.EventType != db.CPUHoursAdd && usageEvent.EventType != db.CPUHoursSubtract {
+		return nil
+	}
```
Comment (psarando) [inline internal/nats.go:44]: Seems like this check makes the `default` case in the `switch` below unreachable.

**Review by psarando — APPROVED** [review]
Looks 🆗 to me 👍

File: internal/nats.go (hunk 1 above)
Comment (slr71) [reply internal/nats.go:44]: Ah, good catch! I neglected to remove this after I added the switch statement.

**Review by johnworth — APPROVED** [review]
LGTM :shipit:

**Pull request merged**